src/site/stages/build/process-cms-exports/index.js

```diff
@@ -106,6 +106,36 @@ const validateOutput = (entity, transformedEntity) => {
   }
 };
 
+/**
+ * Add common properties to the transformed entity. Mutates `transformedEntity`
+ * to save memory.
+ * @param {Object} transformedEntity - The entity after transformation
+ * @param {Object} originalEntity - The entity before transformation
+ * @returns {void}
+ */
+const addCommonProperties = (transformedEntity, originalEntity) => {
+  /* eslint-disable no-param-reassign */
+  transformedEntity.contentModelType =
+    transformedEntity.contentModelType || getContentModelType(originalEntity);
+  const [
+    entityType,
+    entityBundle,
+  ] = transformedEntity.contentModelType.includes('-')
+    ? transformedEntity.contentModelType.split('-')
+    : [transformedEntity.contentModelType, transformedEntity.contentModelType];
```

Comment:
Having `entityType` and `entityBundle` be the same here seems odd.  For things like `paragraph-q_a_section` it's clear which is the bundle and which is the type, but for a plain ol' `paragraph` why do we need the bundle as well?

Reply:
The `entityBundle` is used to determine which Liquid template to use.